src/commands/deploy.ts

```diff
@@ -1,155 +1,77 @@
+        try {
+            cli.action.start(chalk.bold.greenBright('   Checking App Report'));
+            checkReport(this, fd, flags);
+            cli.action.stop(chalk.bold.greenBright('\u2713'));
 
-        cli.action.start(`${ chalk.green('deploying') } your app`);
-
-        const data = new FormData();
-        data.append('app', fs.createReadStream(fd.mergeWithFolder(zipName)));
-
-        await this.asyncSubmitData(data, flags, fd);
-
-        cli.action.stop('deployed!');
-    }
-
-    // tslint:disable:promise-function-async
-    private async asyncSubmitData(data: FormData, flags: { [key: string]: any }, fd: FolderDetails): Promise<void> {
-        let authResult;
-
-        if (!flags.token) {
-            let credentials: { username: string, password: string, code?: string };
-            credentials = { username: flags.username, password: flags.password };
-            if (flags.code) {
-                credentials.code = flags.code;
-            }
-
-            authResult = await fetch(this.normalizeUrl(flags.url, '/api/v1/login'), {
-                method: 'POST',
-                headers: {
-                    'Content-Type': 'application/json',
-                },
-                body: JSON.stringify(credentials),
-            }).then((res: Response) => res.json());
-
-            if (authResult.status === 'error' || !authResult.data) {
-                throw new Error('Invalid username and password or missing 2FA code (if active)');
-            }
-        } else {
-            const verificationResult = await fetch(this.normalizeUrl(flags.url, '/api/v1/me'), {
-                method: 'GET',
-                headers: {
-                    'Content-Type': 'application/json',
-                    'X-Auth-Token': flags.token,
-                    'X-User-Id': flags.userid,
-                },
-            }).then((res: Response) => res.json());
-
-            if (!verificationResult.success) {
-                throw new Error('Invalid API token');
-            }
-
-            authResult = { data: { authToken: flags.token, userId: flags.userid } };
-        }
-
-        let endpoint = '/api/apps';
-        if (flags.update) {
-            endpoint += `/${fd.info.id}`;
-        }
+            cli.action.start(chalk.bold.greenBright('   Getting Server Info'));
+            serverInfo = await getServerInfo(fd, flags);
+            cli.action.stop(chalk.bold.greenBright('\u2713'));
 
-        const deployResult = await fetch(this.normalizeUrl(flags.url, endpoint), {
-            method: 'POST',
-            headers: {
-                'X-Auth-Token': authResult.data.authToken,
-                'X-User-Id': authResult.data.userId,
-            },
-            body: data,
-        }).then((res: Response) => res.json());
+            cli.action.start(chalk.bold.greenBright('   Packaging the app'));
+            zipName = await packageAndZip(this, fd);
+            cli.action.stop(chalk.bold.greenBright('\u2713'));
```

Comment:
```suggestion
            cli.action.start(chalk.bold.greenBright('   Packaging the app'));
            checkReport(this, fd, flags);
            zipName = await packageAndZip(this, fd);
            cli.action.stop(chalk.bold.greenBright('\u2713'));
```

Like this we don't generate extra messages to the developer